relay/src/cli.rs

```diff
@@ -314,6 +319,57 @@ pub fn init_config<P: AsRef<Path>>(config_path: P, _matches: &ArgMatches) -> Res
     Ok(())
 }
 
+/// Manages the on-disk spool file.
+pub fn manage_spool(config: &Config, matches: &ArgMatches) -> Result<()> {
+    if let Some(matches) = matches.subcommand_matches("clear") {
+        let path = match matches.get_one::<String>("path") {
+            Some(path) => {
+                let Ok(path) = PathBuf::from_str(path) else {
+                    bail!("Can't find the provided file path.")
+                };
+                path
+            }
+            None => {
+                let Some(path) = config.spool_envelopes_path() else {
```

Comment:
Can use anyhow's [Context](https://docs.rs/anyhow/latest/anyhow/trait.Context.html):

```suggestion
                let path = config.spool_envelopes_path().context("...")?;
```